weather_dl/download_pipeline/manifest.py

```diff
@@ -96,6 +96,9 @@ class DownloadStatus():
     """The name of the config file associated with the request."""
     config_name: str = ""
 
+    """Represents the dataset field of the configuration."""
```

Comment:
Sanity check: do we need to update any documentation? My instinct is that the answer is "no".

Reply:
You're correct. There is no documentation that needs updating.